recipes/treelite/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "treelite" %}
+{% set version = "0.32" %}
+{% set sha256 = "ac34d0f925dad06d1c90932a57e805e6f0361988f47a457936997fb3aad17cfa" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/dmlc/treelite
+  git_rev: {{ version }}
+
+build:
+  number: 0
+  skip: True  # [win and vc<17]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - setuptools
+    - scipy
```

Comment:
Is `scipy` required to build the package? I don't see it used in the `setup.py`, but it's possible that it's involved

Reply:
This is the `setup` command from the `setup.py`.
It includes `numpy` and `scipy`.
python`setup(
    name='treelite',
    version=VERSION,
    description='treelite: toolbox for decision tree deployment',
    url='http://treelite.io',
    author='DMLC',
    maintainer='Hyunsu Cho',
    maintainer_email='chohyu01@cs.washington.edu',
    packages=find_packages(),
    install_requires=['numpy', 'scipy'],
    package_data={
        'treelite': [LIB_BASENAME, RT_BASENAME, 'treelite_runtime.zip',
                     'treelite_runtime_with_binary.zip', 'VERSION']
    },
    distclass=BinaryDistribution
)`